modules/bluebillywigBidAdapter.js

```diff
@@ -281,11 +281,6 @@ export const spec = {
       tmax: Math.min(BB_CONSTANTS.DEFAULT_TIMEOUT, bidderRequest.timeout),
       imp: imps,
       test: DEV_MODE ? 1 : 0,
-      ext: {
```

Comment:
This looks like it's within a bidder module, I'm not sure this would be relevant to be remove?

Reply:
good point, reverted